### PR TITLE
refactor(css): add styles targeting SVG button/modal icons

### PR DESCRIPTION
**Pull request by TheCatLady**
#### Description

~~Adds and uses global CSS classes `sm-button-icon`, `button-icon`, `button-with-text-icon`, `modal-icon`, and `slider-link-icon`.~~

Applies consistent styles to button and modal icons.

#### To-Dos

- [x] Successful build `yarn build`

#### Issues Fixed or Closed

N/A

### CODE REVIEW

**Comment by sct** [comment]
I think this is more confusing (and actually more to type) than just using the classes directly. Global classes let us change styling site-wide by editing one place, but should we really be doing that in most cases for these SVGs? The context of the SVG's size directly belongs to where it's being used.

**Comment by TheCatLady** [comment]
> I think this is more confusing (and actually more to type) than just using the classes directly. Global classes let us change styling site-wide by editing one place, but should we really be doing that in most cases for these SVGs? The context of the SVG's size directly belongs to where it's being used.

I was thinking that it'd be easier to enforce consistent sizing for certain places where we have icons, which is why these classes don't apply to all SVGs. For example, button icons should all be sized consistently, and the modal icons should always be the same size as well.

**Comment by sct** [comment]
It still means typing out a class for every button icon. A better way of doing this would be creating new classes for buttons that directly target child SVGs and apply the necessary styles.

**Comment by TheCatLady** [comment]
> It still means typing out a class for every button icon. A better way of doing this would be creating new classes for buttons that directly target child SVGs and apply the necessary styles.

Ah, that's very true!  Definitely a better solution.

File: src/components/Common/ButtonWithDropdown/index.tsx [hunk 1]
```diff
@@ -60,24 +60,24 @@ const ButtonWithDropdown: React.FC<ButtonWithDropdownProps> = ({
...
+    dropdownClasses: 'button-md ',
   };
 
   switch (buttonType) {
     case 'ghost':
       styleClasses.mainButtonClasses +=
         ' bg-transparent border-gray-600 hover:border-gray-200 focus:border-gray-100 active:border-gray-100';
       styleClasses.dropdownSideButtonClasses = styleClasses.mainButtonClasses;
-      styleClasses.dropdownClasses = 'bg-gray-700';
+      styleClasses.dropdownClasses += 'bg-gray-700';
```
Comment (sct) [inline src/components/Common/ButtonWithDropdown/index.tsx:73]: why not prepend the space here as we do for the other classes?

File: src/styles/globals.css [hunk 2]
```diff
@@ -281,6 +285,23 @@ select.short {
   @apply w-min;
 }
 
+button svg,
+.button-md svg {
+  @apply w-5 h-5 mr-2 last:mr-0;
+}
+
+.button-sm svg {
+  @apply w-4 h-4 mr-1.5 last:w-5 last:h-5 last:mr-0;
```
Comment (sct) [inline src/styles/globals.css:294]: Why is the icon larger only if it's the last item?

Comment (TheCatLady) [reply src/styles/globals.css:294]: We use 5x5 icons along with the smaller button size when there is no accompanying text (e.g., logs page). It's not so much that it's the last item, but the only item.

**Review by sct — APPROVED** [review]

**Pull request merged**